### DISCUSSION TITLE
svtr训练速度比resnet34 慢很多？同等配置，有没有办法加速？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12591] [Q&A] [answered]
#### 运行环境 / Runtime Environment
- OS:
- Paddle: 2.3
- PaddleOCR: 2.7.5

#### 附件 / Appendix

svtr训练速度比resnet34 慢很多？同等配置，有没有办法加速？

**Comment by GreatV** [comment, marked as answer]
没有
